ansible/roles/docker/tasks/main.yml

```diff
@@ -0,0 +1,120 @@
+---
+# tasks file for role/docker
+- name: Upgrade all apt packages
+  apt: upgrade=dist force_apt_get=yes
+
+- name: Create a login user
+  user:
+    name: "{{ user }}"
+    state: present
+
+- name: Add public key to authorized_keys
+  authorized_key:
+    user: "{{ user }}"
+    state: present
+    key: "{{ lookup('file', '~/sammy.pub') }}"
```

Comment:
what is this file?

Reply:
Changing it now to the default public key file-name